### ISSUE TITLE
Phone Headset Wiring

### ISSUE THREAD

**Issue by ChrishaJCG** [nickpourazima/rotary-phone-audio-guestbook#35]
Hello, thank you for making this project, I am attempting to recreate it with the hardware that I have but I am running into an issue. I purchased a usb audio adapter but from the phone I will only ever get the speaker to be detected. Could you show the wiring from the handset jack to the usb audio adapter?

Thank you!

**Comment by nickpourazima** [comment]
Hi @ChrishaJCG,

Sure. I took a few more pictures:

The USB audio adapter typically accepts 2 x 3.5mm male audio jacks which goes tip/ring/sleeve (TRS). I soldered my wires directly to the board but it isn't necessary for this project. This is what the top of my audio card looks like, note the differentiation between the microphone input and the speaker output:
![AudioCardTopSide](https://github.com/nickpourazima/rotary-phone-audio-guestbook/assets/12181252/d9b56dbc-daa5-4720-ba41-2228a3fc8f89)

In general a TRS jack maps to:
Tip (T) --> mic signal 1 (we'll call this positive M+)
Ring (R) --> mic signal 2 (we'll call this negative M-)
Sleeve (S) --> GND
![TRS_AudioCard](https://github.com/nickpourazima/rotary-phone-audio-guestbook/assets/12181252/517f075f-80fb-4c95-b456-41084b200bf6)

And similarly for the speaker, T -> S+, R -> S-, S -> GND as in the picture:
![image](https://github.com/nickpourazima/rotary-phone-audio-guestbook/assets/12181252/2576a6cb-d032-4022-ade8-9aa15500fdda)

Here is the mapping from the phone jack (left) to USB Audio Card (right):
Black --> Mic S+
Red --> Mic S- (or vis-versa, doesn't really matter)
Green --> GND
White --> Speaker (+)
And I left Speaker (-) disconnected since it wasn't necessary.

![AudioCardToPhoneJack](https://github.com/nickpourazima/rotary-phone-audio-guestbook/assets/12181252/854d7c3d-d363-4fd7-9852-0dbc9c8a725c)

One thing to confirm from your side is whether your 3.5mm jack is TRS since it could be a TRRS which in that case an extra Ring is present (typically for headsets with built in microphones to carry all 3 signals (left audio, right audio, mic).
